flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -898,13 +873,6 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
-		<dependency>
-			<groupId>org.apache.hadoop</groupId>
-			<artifactId>hadoop-hdfs</artifactId>
```

Comment:
This shouldn't have been removed; but instead the version should only have been adjusted.

Reply:
Yeah I noticed, focussed too much on the failing Kafka test. I've adjusted the changes